packages/dev/docs/pages/releases/2023-12-18.mdx

```diff
@@ -0,0 +1,128 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we’re promoting React Aria Components from RC to GA 🎉, optimizing our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
+
+date: 2023-12-18
+---
+
+# December 18, 2023 Release
+
+Welcome to the last release of the year! In this release, we’re promoting React Aria Components from RC to GA 🎉, promoting React Spectrum's Dropzone to RC, adding an option to optimize our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
+
+As always, huge thanks to everyone in our community for their feedback and contributions to make this release and this year possible!
+
+In a big year, there are a few accomplishments we wanted to highlight.
+ - ESM support
+ - React StrictMode support
+ - React Aria Components
+ - React Spectrum new components, Dropzone, InlineAlert, TagGroup, ActionBar, Avatar, Table Col Resizing
+ - Docs Search
+ - Built-in support for form validation
+ - Native links support for collections
+ - Tailwind plugin for RAC
+
+As a parting holiday gift, check out our new landing page for [React Aria Components](../react-aria/index.html), replete with interactive examples! Happy coding and see you in the new year! ☃️
+
+
+### React Aria Components API updates
+
+In this release, we made a few last API updates to React Aria Components as they move along to GA.
+
+We had to temporarily remove nested column support in Tables due to some last minute use cases which we deemed to be inconsistent. We wouldn't have found these without all of your help using our components. Nested column support will be back, and should be easier to use in the future. Thanks for understanding.
```

Comment:
I would shorten this to just like "Temporarily removed nested column support in Tables due to instability." Not sure we need to explain ourselves all that much in the release notes.